File: Python/RAM.py
class RAM:
    #atribut
    __capacity=0
    __price=0

    #constructor
    def __init__(self, capacity=0, price=0):
        self.__capacity=capacity
        self.__price=price

    #Setter Getter
    def setCapacity(self, capacity):
        self.capacity=capacity

    def setPrice(self, price):
        self.__price=price
    
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price

    #Method
    #Menampilkan atribut RAM
    def printRAM(self):
        print("RAM Capacity: " + str(self.getCapacity()))
        print("RAM Price: " + str(self.getPrice()))

File: Java/Ram.java
public class Ram {
    //atribut
    private int capacity;
    private int price;

    //constructor
    Ram() {}
    Ram(int capacity, int price) {
        this.capacity=capacity;
        this.price=price;
    }

    //Setter Getter
    public void setCapacity(int capacity) { this.capacity=capacity; }
    public void setPrice(int price) { this.price=price; }
    
    public int getCapacity() { return this.capacity; }
    public int getPrice() { return this.price; }

    //Method
    //Menampilkan atribut RAM
    public void printRam() {
        System.out.println("RAM Capacity : " + this.getCapacity());
        System.out.println("RAM Price : " + this.getPrice());
    }
}
